modular_skyrat/master_files/code/modules/clothing/under/akula_official.dm

```diff
@@ -0,0 +1,35 @@
+/obj/item/clothing/under/rank/azulean
+	icon_state = ""
```

Comment:
Starting it off with an empty icon state like this is generally bad practice, and it looks like it was just forgotten.